font/README.md

```diff
@@ -1,3 +1,4 @@
-These fonts have been generated with
+These fonts have been generated with:
+
 	makefont.sh 8 16 12 SourceCodePro-Bold.ttf font.bin 
```

Comment:
Please also add code clock via "```" for the two makefont.sh lines

Reply:
The indentation alone makes it format as code.
If I add the code block (```) then it includes the indentation along with the code.